ArcGISRuntimeSDKQt_QMLSamples/Scenes/ScenePropertiesExpressions/ScenePropertiesExpressions.qml

```diff
@@ -0,0 +1,184 @@
+// [WriteFile Name=ScenePropertiesExpressions, Category=Scenes]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.5
+import QtQuick.Controls 2.2
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property real longitude: -118.71
+    property real latitude: 32.09
+    property real altitude: 100000.0
+    property int coneDimension: 10000
+    property real initialPitch: 90.0
+    property real initialHeading: 180.0
+
+    SceneView {
+        id: sceneView
+        anchors.fill: parent
+
+        Scene {
+            id: scene
+            BasemapImagery {}
+
+            Surface {
+                ArcGISTiledElevationSource {
+                    url: "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+                }
+            }
+        }
+
+        // add a graphics overlay
+        GraphicsOverlay {
+            id: graphicsOverlay
+            LayerSceneProperties {
+                surfacePlacement: Enums.SurfacePlacementAbsolute
+            }
+
+            SimpleRenderer {
+                id: sceneRenderer
+                RendererSceneProperties {
+                    id: renderProps
+                    headingExpression: "[Heading]"
+                    pitchExpression:  "[Pitch]"
+                }
+            }
+
+            Graphic {
+                id: coneGraphic
+
+                geometry: Point {
+                    x: longitude
+                    y: latitude
+                    z: altitude
+                    spatialReference: sceneView.spatialReference
+                }
+
+                SimpleMarkerSceneSymbol{
+                    style: Enums.SimpleMarkerSceneSymbolStyleCone
+                    color: "red"
+                    width: coneDimension
+                    depth: coneDimension
+                    height: coneDimension * 2
+                    anchorPosition: Enums.SceneSymbolAnchorPositionCenter
+                }
+
+                Component.onCompleted: {
+                    coneGraphic.attributes.insertAttribute("Heading", initialHeading);
```

Comment:
you could centralize the attribute names as a property

Reply:
Done, added string properties.